api/lib/domain/models/CertificationPartnerClea.js

```diff
@@ -0,0 +1,83 @@
+const _ = require('lodash');
+const Badge = require('../models/Badge');
+const CertificationPartner = require('./CertificationPartner');
+const MIN_PERCENTAGE = 75;
+
+const { MINIMUM_REPRODUCIBILITY_RATE_TO_BE_CERTIFIED, MINIMUM_REPRODUCIBILITY_RATE_TO_BE_TRUSTED } = require('../constants');
+
+function _isOverPercentage(value = 0, total, percentage = MIN_PERCENTAGE) {
+  return value >= (total * percentage / 100);
+}
+
+function _hasRequiredPixValue({ totalPixCleaByCompetence, competenceMarks }) {
+  const certifiableCompetenceIds = _.map(competenceMarks, 'competenceId');
+  const partnerCompetenceIds = _.keys(totalPixCleaByCompetence);
+  const consideredCompetenceIds = _.intersection(certifiableCompetenceIds, partnerCompetenceIds);
+  return !_.isEmpty(consideredCompetenceIds)
+    && _.every(consideredCompetenceIds, (competenceId) => _isOverPercentage(
+      _.find(competenceMarks, { competenceId }).score,
+      totalPixCleaByCompetence[competenceId]
+    ));
+}
+
+function _hasSufficientReproducibilityRateToBeCertified(reproducibilityRate) {
+  return reproducibilityRate >= MINIMUM_REPRODUCIBILITY_RATE_TO_BE_TRUSTED;
+}
+
+function _hasNotMinimumReproducibilityRateToBeTrusted(reproducibilityRate) {
+  return reproducibilityRate <= MINIMUM_REPRODUCIBILITY_RATE_TO_BE_CERTIFIED;
+}
+
+class CertificationPartnerClea extends CertificationPartner {
+
+  constructor({
+    certificationCourseId,
+    hasAcquiredBadgeClea,
+    reproducibilityRate,
+    competenceMarks,
+    totalPixCleaByCompetence
+  } = {}) {
+    super({
+      certificationCourseId,
+      partnerKey: Badge.keys.PIX_EMPLOI_CLEA,
+    });
+
+    this.hasAcquiredBadge = hasAcquiredBadgeClea;
+    this.reproducibilityRate = reproducibilityRate;
+    this.competenceMarks = competenceMarks;
+    this.totalPixCleaByCompetence = totalPixCleaByCompetence;
+  }
+
+  isEligible() {
+    return this.hasAcquiredBadge;
+  }
+
+  isAcquired() {
+    if (_hasNotMinimumReproducibilityRateToBeTrusted(this.reproducibilityRate)) return false;
+
+    if (_hasSufficientReproducibilityRateToBeCertified(this.reproducibilityRate)) return true;
+
+    return _hasRequiredPixValue({
+      competenceMarks: this.competenceMarks,
+      totalPixCleaByCompetence:this.totalPixCleaByCompetence
+    });
+  }
```

Comment:
Remarque en l'air (avec ma méconnaissance de javascript), je sais pas si ça marche : on veut des méthodes privées mais on peut pas donc on fais des méthodes hors de la classe qu'on exporte pas. Est-ce qu'au lieu de faire du passe-plat d'argument on peut pas plutôt faire `_hasRequiredPixValue().bind(this)` et utiliser `this.competenceMarks` dans `_hasRequiredPixValue()` ? Si ça marche ça me semblera plus proche de méthodes privées tout en gardant le coté non-accessible depuis l'exterieur non ?